src/sockperf.cpp

```diff
@@ -3367,10 +3367,9 @@ int bringup(const int *p_daemonize) {
                         s_user_params.client_bind_info.ss_family = AF_UNIX;
                         if (strlen(s_user_params.client_bind_info.addr_un.sun_path) == 0 && s_user_params.mode == MODE_CLIENT) { // no specific addr client_info was provoided
                             log_dbg("No client name was provided, setting addr_un.sun_path to %s_client\n",s_user_params.addr.addr_un.sun_path);
-                            char result[sizeof(s_user_params.addr.addr_un.sun_path) + sizeof("_client")];
-                            strcpy(result,s_user_params.addr.addr_un.sun_path);
-                            strcat(result,"_client");
-                            strcpy(s_user_params.client_bind_info.addr_un.sun_path, result);
+                            std::string result = s_user_params.addr.addr_un.sun_path;
+                            result += "_client";
+                            strcpy(s_user_params.client_bind_info.addr_un.sun_path, result.c_str());
```

Comment:
Are you sure that you have enough memory in destination array?

Reply:
sun_path size limit is 108 
https://man7.org/linux/man-pages/man7/unix.7.html

Reply:
108 + strlen("_client") > 108